src/components/ProposalList/ProposalListItem.tsx

```diff
@@ -1,58 +1,55 @@
-import { Avatar, AvatarImage, AvatarFallback } from '@radix-ui/react-avatar';
 import { Label } from '@radix-ui/react-label';
 import { RadioGroup, RadioGroupItem } from '@radix-ui/react-radio-group';
 import { ThumbsUpIcon, ThumbsDownIcon, MinusIcon } from 'lucide-react';
-import { CardHeader, CardContent, Card } from './ui/card.tsx';
+import {
+  CardHeader,
+  CardContent,
+  Card,
+  CardTitle,
+  CardDescription,
+} from '../ui/card.tsx';
+import type { Proposal } from '../../types/proposal.ts';
 
-export type ProposalVoteCardProps = {
-  author?: string;
-  title: string;
-  description: string;
-  created: string;
+export type Props = {
+  proposal: Proposal;
 };
 
-export function ProposalVoteCard(props: ProposalVoteCardProps) {
+export default function ProposalListItem({ proposal }: Props) {
   return (
     <Card>
       <CardHeader className="pb-0 pt-6">
-        <div className="flex items-center justify-between">
-          <div>
-            <h3 className="text-lg font-semibold dark:text-gray-200">
-              {props.title}
-            </h3>
-            <p className="text-gray-500 dark:text-gray-400">
-              {props.description}
-            </p>
-          </div>
+        <CardTitle className="flex justify-between">
+          {proposal.title}
+
           <div className="flex items-center gap-2">
             <ThumbsUpIcon className="w-5 h-5 text-green-500" />
             <span className="text-green-500 font-medium">72</span>
             <ThumbsDownIcon className="w-5 h-5 text-red-500" />
             <span className="text-red-500 font-medium">18</span>
           </div>
-        </div>
+        </CardTitle>
+        <CardDescription>
+          Proposed{' '}
+          {new Date(proposal.created).toLocaleDateString('en-US', {
+            month: 'long',
+            day: 'numeric',
+            year: 'numeric',
+          })}{' '}
+          at{' '}
+          {new Date(proposal.created).toLocaleTimeString('en-US', {
+            hour: 'numeric',
+            minute: 'numeric',
+            hour12: true,
+          })}{' '}
+          by{' '}
+          <span className="font-bold dark:text-gray-200">
+            {proposal.authorName ?? 'Anonymous'}
+          </span>
+        </CardDescription>
+        <CardDescription>{proposal.description}</CardDescription>
```

Comment:
ISSUE: The description probably should be part of the CardContent
ISSUE: While I agree that the `ProposedBy` line should probably be part of the description, I'm not a huge fan of having two <CardDescription> tags...  but putting them in 1 makes the styling ugly.
SUGGESTION: Let's move `description` into `CardContent` and remove the `<CardDescription>` wrapper around the `proposed-by` section, for now
```suggestion
 <CardContent className="flex flex-col items-start justify-between gap-6">
        <div className="flex items-center gap-1 py-0">
          <Avatar className="w-8 h-8 content-center text-center">
            <AvatarImage alt="@shadcn" src="/placeholder-user.jpg" />
            <AvatarFallback>JD</AvatarFallback>
          </Avatar>
          <div className="text-sm text-gray-500 dark:text-gray-400">
            {`Proposed ${new Date(proposal.created).toLocaleDateString(
              'en-US',
              {
                month: 'long',
                day: 'numeric',
                year: 'numeric',
              },
            )} at ${new Date(proposal.created).toLocaleTimeString('en-US', {
              hour: 'numeric',
              minute: 'numeric',
              hour12: true,
            })} by `}
            <span className="font-bold dark:text-gray-200">
              {proposal.authorName ?? 'Anonymous'}
            </span>
          </div>
        </div>
        <CardDescription className="text-gray-500 dark:text-gray-400">
          {proposal.description}
        </CardDescription>
```

Reply:
Sounds good. I've moved the content around per feedback.